src/puppetlabs/puppetdb/utils/logging.clj

```diff
@@ -0,0 +1,57 @@
+(ns puppetlabs.puppetdb.utils.logging
```

Comment:
Any discussions with people, where this might land if it was an okay pattern to repeat for other projects? I was actually considering this kind of thing for hafjell the other day (I swear) and was surprised we didn't already have logging wrappers like this :-).